src/schemathesis/extra/pytest_plugin.py

```diff
@@ -36,7 +38,10 @@ def _gen_items(self, endpoint: Endpoint) -> Generator[Function, None, None]:
         Could produce more than one test item if
         parametrization is applied via ``pytest.mark.parametrize`` or ``pytest_generate_tests``.
         """
-        hypothesis_item = create_test(endpoint, self.test_function)
+        try:
+            hypothesis_item = create_test(endpoint, self.test_function)
+        except InvalidEndpoint:
+            pytest.fail("Invalid endpoint")
```

Comment:
It would be nice to have some more verbose error definition, e.g. that there is an error in the schema + maybe some info about from the underlaying exception